server/datastore/mysql/migrations/tables/20220201144450_AddSoftwareReleaseArchVendorColumns.go

```diff
@@ -0,0 +1,61 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220201144450, Down_20220201144450)
+}
+
+func Up_20220201144450(tx *sql.Tx) error {
```

Comment:
This could be an interesting chance to add a test for this, a-la this: https://github.com/fleetdm/fleet/commit/37a78eebd79023b8329a2cc7283d6ec9f5d534e1